### ISSUE TITLE
[bug] The CLI is not running from the dist folder.

### ISSUE THREAD

**Issue by buhodev** [inlang/monorepo#1478]
### Problem

When I try to compile paraglide-js in my project as a prebuild step, I get this error:

```shell
This is likely an internal bug. Please file an issue at https://github.com/inlang/monorepo/issues.

Debug information:

- metaUrlPath: C:\Users\buhodev\Projects\svelte-one
ode_modules\@inlang\paraglide-js\dist\cli\main.js
```

I thought it was due to pnpm, but I tried with npm and it didn't work either.

Also, I tried editing manually the `metaUrlPath` to be `/Users/buhodev/Projects/svelte-one/node_modules/@inlang/paraglide-js/dist/cli/main.js` but more errors appeared:

```shell
ℹ Compiling inlang project at "./project.inlang.json".                                                                                        9:56:06 PM
file:///C:/Users/buhodev/Projects/svelte-one/node_modules/@inlang/sdk/dist/loadProject.js:30
        throw new LoadProjectInvalidArgument(`Expected an absolute path but received "${args.settingsFilePath}".`, { argument: "settingsFilePath" });      
              ^

LoadProjectInvalidArgument: The argument "settingsFilePath" of loadProject() is invalid: Expected an absolute path but received "C:\Users\buhodev\Projects\svelte-one\project.inlang.json".
    at loadProject (file:///C:/Users/buhodev/Projects/svelte-one/node_modules/@inlang/sdk/dist/loadProject.js:30:15)
    at runCompileCommand (file:///C:/Users/buhodev/Projects/svelte-one/node_modules/@inlang/paraglide-js/dist/cli/commands/compile.js:27:40)
    at Command.<anonymous> (file:///C:/Users/buhodev/Projects/svelte-one/node_modules/@inlang/paraglide-js/dist/cli/commands/compile.js:19:5)
    at Command.listener [as _actionHandler] (C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:494:17)
    at C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:1296:65
    at Command._chainOrCall (C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:1193:12)
    at Command._parseCommand (C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:1296:27)
    at C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:1082:27
    at Command._chainOrCall (C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:1193:12)
    at Command._dispatchSubcommand (C:\Users\buhodev\Projects\svelte-one\node_modules\commander\lib\command.js:1078:25)

Node.js v18.15.0
 ELIFECYCLE  Command failed with exit code 1.
```

### Expected behavior

To compile paraglide-js

### Reproduction

1. Download the sveltekit example folder from https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/inlang/monorepo/tree/main/inlang/source-code/paraglide/paraglide-js-adapter-sveltekit/example
2. Install dependencies
3. Add `https://cdn.jsdelivr.net/npm/@inlang/plugin-message-format@latest/dist/index.js` to the modules array in the `project.inlang.json` file
4. Run `build` command
5. See error

### Other information

OS: Windows 11
Node.js v18.15.0

**Comment by samuelstroschein** [comment]
@felixhaeberle can you fix this?

**Comment by doic** [comment]
Having the same issue, waiting for the fix 🤍

**Comment by felixhaeberle** [comment]
@doic @buhodev Pipelines are currently running for the new release. 🚀 
Please make sure to run on the latest dependencies. Even if we do not run on Windows ourselves, we wrote a test to catch this error. If there is any error left, please reopen this issue.

**Comment by buhodev** [comment]
Now the first error disappear, but I get the second error:

```shell
> paraglide-js compile --namespace svelte-one

ℹ Compiling inlang project at "./project.inlang.json".
file:///C:/Users/buhodev/Projects/svelte-one/node_modules/.pnpm/@inlang+sdk@0.13.0/node_modules/@inlang/sdk/dist/loadProject.js:30
        throw new LoadProjectInvalidArgument(`Expected an absolute path but received "${args.settingsFilePath}".`, { argument: "settingsFilePath" });
              ^

LoadProjectInvalidArgument: The argument "settingsFilePath" of loadProject() is invalid: Expected an absolute path but received "C:\Users\buhodev\Projects\svelte-one\project.inlang.json".
    at loadProject (file:///C:/Users/buhodev/Projects/svelte-one/node_modules/.pnpm/@inlang+sdk@0.13.0/node_modules/@inlang/sdk/dist/loadProject.js:30:15)
    at runCompileCommand (file:///C:/Users/buhodev/Projects/svelte-one/node_modules/.pnpm/@inlang+paraglide-js@1.0.0-prerelease.3/node_modules/@inlang/paraglide-js/dist/cli/commands/compile.js:27:40)
    at Command.<anonymous> (file:///C:/Users/buhodev/Projects/svelte-one/node_modules/.pnpm/@inlang+paraglide-js@1.0.0-prerelease.3/node_modules/@inlang/paraglide-js/dist/cli/commands/compile.js:19:5)
    at Command.listener [as _actionHandler] (C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:494:17)        
    at C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:1296:65
    at Command._chainOrCall (C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:1193:12)
    at Command._parseCommand (C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:1296:27)
    at C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:1082:27
    at Command._chainOrCall (C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:1193:12)
    at Command._dispatchSubcommand (C:\Users\buhodev\Projects\svelte-one\node_modules\.pnpm\commander@11.1.0\node_modules\commander\lib\command.js:1078:25)

Node.js v18.15.0
 ELIFECYCLE  Command failed with exit code 1.
```

I deleted `node_modules`, and installed fresh the latest version of paraglide-js (`@inlang/paraglide-js@1.0.0-prerelease.3`)

**Comment by samuelstroschein** [comment]
@buhodev thanks for reporting. 

@felixhaeberle the bug seems to be this line https://github.com/inlang/monorepo/commit/6ddaffaeb636225184e0141aab78a814bd3ce489#r130203730 where isAbsolutePath is still not working. 

I propose to change all fs logic to use POSIX paths and ensure that the entry is normalizing. So instead of the regex stuff in `isAboslutePath`, `loadProject` immediately normalizes the project settings file path. The rest of the system assumes POSIX.

**Comment by felixhaeberle** [comment]
I don't understand why this is happening, it should resolve true as seen in the tests

---

For your proposal: You won't prevent the regex in `isAbsolutePath` tho but only make the scope smaller to only match forward slashes so the complexity with the drive letter & more for windows still persist.

We already normalizing the path before entering `createNodeishFsWithAbsolutePaths` - but feel free to put it even before the validation of the `settingsFilePath`.

**Comment by samuelstroschein** [comment]
@buhodev we found the error. NPM workspaces don't pin the latest version. 

Your package-lock resolves the SDK 0.13 which does not contain the windows fix which landed in 0.14. Can you delete your package lock? 

<img width="558" alt="CleanShot 2023-10-17 at 17 46 02@2x" src="https://github.com/inlang/monorepo/assets/35429197/6d6aeb5b-4474-4e40-88ad-20dfde54ed9b">

We will try to find a solution for pinning versions.

**Comment by buhodev** [comment]
Solved! 🎉
